lib/ruy/conditions/assert.rb

```diff
@@ -1,20 +1,30 @@
 module Ruy
   module Conditions
 
-    # Asserts that an attribute has a truthy value.
+    # Expects that the context key stores a truthy value
+    #
     class Assert < Condition
-      attr_reader :attr
+      attr_reader :key
 
-      # @param attr Context attribute's attr
-      def initialize(*attrs)
+      # @param key
+      # @example evaluate that :key passes
+      #   Asset.new(:key)
+      def initialize(*keys)
         super
-        @attr = attrs.first if attrs.any?
+        @key = keys.first if keys.any?
```

Comment:
I believe if keys.any? shouldn't be necessary. @key will be nil unless keys.any? anyway.